lib/linkedin_scraper/profile.rb

```diff
@@ -77,7 +77,7 @@ def summary
     end
 
     def picture
-      @picture ||= (@page.at(".profile-picture img").attributes["src"].value.strip if @page.at(".profile-picture img"))
+      @picture ||= (@page.at('.profile-picture img').attributes.values_at('src','data-delayed-url').compact.first.value.strip if @page.at('.profile-picture img'))
```

Comment:
Do we need `rescue nil` just to make sure no method call is made on nil object ?